pegasus/helpers/section_api_helpers.rb

```diff
@@ -485,6 +485,7 @@ def script
   def to_owner_hash
     to_member_hash.merge(
       script: script,
+      course_id: @row[:course_id],
```

Comment:
Teacher-dashboard does a bunch of stuff with the script it gets back (for example, iterating through the stages in it). For what we have planned with courses, I don't think we should need any of that - we just need to know the id of the course the section is assigned to. As such, it seems silly to return any more data than needed.